frontend/src/components/translation/SendForReviewModal.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Text,
+} from "@chakra-ui/react";
+
+type SendForReviewModalProps = {
+  sendForReview: boolean;
+  onSendForReviewClick: () => void;
+  onSendForReviewConfirmationClick: () => void;
+};
+
+export const SendForReviewModal = ({
```

Comment:
no need for the `export` keyword as this component is exported as the default item for this file at the bottom of this file (`export default`)